R/sgdnet.R

```diff
@@ -261,6 +265,15 @@ sgdnet.default <- function(x,
 
   if (maxit <= 0)
     stop("maximum number of iterations cannot be negative or zero.")
+  
+  if (batchsize <= 0)
+    stop("batch size cannot be negative or zero.")
+  
+  if (batchsize > n_samples)
+    stop("batch size cannot be larger than sample size.")
+  
+  if (batchsize%%1 > 0)
```

Comment:
Maybe use the `is.wholenumber` function given in `?is.integer` here - this check is a bit opaque to me.

Reply:
Hi, I am using the method from is.wholenumber instead.